nbextensions/advancedViewCell/widgets/appParamsWidget.js

```diff
@@ -34,19 +34,17 @@ define([
         div = t('div');
 
     function factory(config) {
-        let runtime = Runtime.make(),
+        const runtime = Runtime.make(),
```

Comment:
misc eslint fixes, added classes to the toggles, and swapped out `name: name` for `name`